geomstats/_backend/tensorflow/__init__.py

```diff
@@ -354,6 +356,10 @@ def array_from_sparse(indices, data, target_shape):
         tf.SparseTensor(indices, data, target_shape)))
 
 
+def get_slice(x, indices):
+    return gather_nd(x, list(zip(*indices)))
```

Comment:
Ah, I think it's better to have `tf.gather_nd`... here instead of importing the function explicitly. I prefer for the import list at the beginning of the file to contain only functions which are exported or aliased without change.

Reply:
That sounds sensible, I'm on it.